<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>6</y>
     <width>180</width>
     <height>338</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Settings</string>
    </property>
   </column>
   <item>
    <property name="text">
     <string>Allgemein</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Punkte In/Export</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Profilentzerrung</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Grabungsfotos</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Projekt</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Messen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Listenausgabe</string>
    </property>
    <item>
     <property name="text">
      <string>Befund</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fund</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Profil</string>
     </property>
    </item>
   </item>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>197</x>
     <y>5</y>
     <width>426</width>
     <height>341</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Allgemein</string>
    </attribute>
    <widget class="QCheckBox" name="chbautoSave">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>34</y>
       <width>58</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>An</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>36</y>
       <width>34</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>min</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtautoSaveTime">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>33</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>14</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Automatische Speicherung</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>87</y>
       <width>105</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Textgröße</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="spbTextGr">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>67</y>
       <width>270</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Textgröße bei anderer Bildauflösung</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Punkte In/Export</string>
    </attribute>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>9</y>
       <width>194</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPointPfadImp">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>49</y>
       <width>400</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>194</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pfad zum Importordner</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>122</y>
       <width>194</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pfad zum Exportordner</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPointPfadExp">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>141</y>
       <width>400</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>101</y>
       <width>194</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Profilentzerrung</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>10</y>
       <width>403</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Einstellungen für den Export der Fotoentzerrpunkte bei Profilen</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtfeldNProfNr">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>54</y>
       <width>187</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>211</x>
       <y>56</y>
       <width>194</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Feldname in der die Profilnummer steht</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtFeldNFEP">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>90</y>
       <width>187</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>194</width>
       <height>24</height>
      </rect>
     </property>
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string>Feldname in der die Entzerrpunkt-definition steht</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtAttFEP">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>124</y>
       <width>187</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>212</x>
       <y>126</y>
       <width>194</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Attribut Entzerrpunkt</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtProfilFEPPfadExp">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>174</y>
       <width>400</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>155</y>
       <width>194</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pfad zum Exportordner</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Grabungsfotos</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>403</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Bildpfade der Grabungsfotos</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtautoSave_5">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>51</y>
       <width>403</width>
       <height>39</height>
      </rect>
     </property>
     <property name="text">
      <string>Noch nicht in gebrauch</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>194</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pfad org. Grabungsfotos</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>101</y>
       <width>194</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pfad Thumbs Grabungsfotos</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtautoSave_6">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>117</y>
       <width>403</width>
       <height>39</height>
      </rect>
     </property>
     <property name="text">
      <string>Noch nicht in gebrauch</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Projekt</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>10</y>
       <width>403</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Projekteinstellungen</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>31</y>
       <width>82</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Aktivität</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtautoSave_7">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>47</y>
       <width>219</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Noch nicht in gebrauch</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtautoSave_8">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>87</y>
       <width>219</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Noch nicht in gebrauch</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>71</y>
       <width>194</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Gemarkung</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtautoSave_9">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>130</y>
       <width>219</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Noch nicht in gebrauch</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>114</y>
       <width>194</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Gemeinde</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtautoSave_10">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>172</y>
       <width>219</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Noch nicht in gebrauch</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>156</y>
       <width>194</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Projektname</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Messen</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>16</y>
       <width>88</width>
       <height>249</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="chbMInfo_4">
        <property name="text">
         <string>Geometrieart</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbMInfo_5">
        <property name="text">
         <string>Objekttyp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbMInfo_6">
        <property name="text">
         <string>Objektart</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbMInfo_7">
        <property name="text">
         <string>Schnitt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbMInfo_8">
        <property name="text">
         <string>Planum</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbMInfo_9">
        <property name="text">
         <string>Befund</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbMInfo_10">
        <property name="text">
         <string>Fund</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbMInfo_11">
        <property name="text">
         <string>Profil</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbMInfo_12">
        <property name="text">
         <string>Probe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbMInfo_13">
        <property name="text">
         <string>Punkt Nr</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbMInfo_14">
        <property name="text">
         <string>Koordinaten</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>16</y>
       <width>132</width>
       <height>65</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="chbMInfo_1">
        <property name="text">
         <string>Mausinfo anzeigen</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbMInfo_2">
        <property name="text">
         <string>bei Begin der Messung</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbMInfo_3">
        <property name="text">
         <string>immer anzeigen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="butOK">
   <property name="geometry">
    <rect>
     <x>546</x>
     <y>358</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butAbruch">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>358</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Abbruch</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
